spec/models/importers/single_content_item_spec.rb

```diff
@@ -1,105 +1,105 @@
-        ).pluck(:target_content_id)
-      ).to contain_exactly('org-id-1', 'org-id-2')
+        content_item = ContentItem.last
 
-      expect(
-        content_item_links.where(
-          link_type: "taxons"
-        ).pluck(:target_content_id)
-      ).to contain_exactly('taxon-id-1', 'taxon-id-2')
-    end
-  end
-
-  context "when the content item already exists" do
-    let(:updated_content_item) { build(:content_item, content_id: content_id, title: "new title") }
+        expect(content_item.title).to eq("title")
+        expect(content_item.linked_organisations).to match_array([org1, org2])
+        expect(content_item.linked_taxons).to match_array([taxon1, taxon2])
+        expect(content_item.number_of_pdfs).to eq(10)
+      end
 
-    before do
-      allow(subject.content_items_service).to receive(:fetch).with(content_id).and_return(updated_content_item)
+      it "imports links" do
+        expect { subject.run(content_id) }
+          .to change(Link, :count).by(4)
+
+        content_item = ContentItem.last
+        content_item_links = Link.where(source_content_id: content_item.content_id)
+
+        expect(
+          content_item_links.where(
+            link_type: "organisations"
+          ).pluck(:target_content_id)
+        ).to contain_exactly('org-id-1', 'org-id-2')
+
+        expect(
+          content_item_links.where(
+            link_type: "taxons"
+          ).pluck(:target_content_id)
+        ).to contain_exactly('taxon-id-1', 'taxon-id-2')
+      end
     end
 
-    it "updates the content item" do
-      content_item.save!
+    context "when the content item already exists" do
+      let(:updated_content_item) { build(:content_item, content_id: content_id, title: "new title") }
 
-      expect { subject.run(content_id) }
-        .to change { content_item.reload.title }.from("title").to("new title")
-        .and change { content_item.reload.linked_taxons.to_a }.from([]).to([taxon1, taxon2])
-        .and change { content_item.reload.linked_organisations.to_a }.from([]).to([org1, org2])
-        .and change { content_item.reload.number_of_pdfs }.from(0).to(10)
-    end
-  end
+      before do
+        allow(subject.content_items_service).to receive(:fetch).with(content_id).and_return(updated_content_item)
+      end
 
-  context "when the links already exists, and the Publishing API is unchanged" do
-    before do
-      allow(subject.content_items_service).to receive(:fetch).with(content_id).and_return(content_item)
-      subject.run(content_id)
-    end
+      it "updates the content item" do
+        content_item.save!
 
-    it "doesn't create any additional links" do
-      expect { subject.run(content_id) }
-        .not_to change(Link, :count)
+        expect { subject.run(content_id) }
+          .to change { content_item.reload.title }.from("title").to("new title")
+                .and change { content_item.reload.linked_taxons.to_a }.from([]).to([taxon1, taxon2])
```

Comment:
I think I find this slightly harder to read. Is it a convention to 
```
.chain
  .it
    .like
      .this
```
as opposed to

```
.chain
.it
.like
.this
```
?

It's already clear to me that as each line starts with a `.` that it's a continuation of method chaining without the additional indentation.